src/main/java/uk/gov/justice/digital/hmpps/keyworker/services/NomisService.java

```diff
@@ -22,11 +21,11 @@
     String URI_CURRENT_ALLOCATIONS_BY_OFFENDERS = "/key-worker/{agencyId}/current-allocations/offenders";
     String URI_OFFENDERS_ALLOCATION_HISTORY = "/key-worker/offenders/allocationHistory";
     String URI_GET_ALL_PRISONS = "/agencies/prison";
-    String URI_GET_AGENCY = "/agencies/{agencyId}?activeOnly=false&agencyType={agencyType}";
+    String URI_GET_AGENCY = "/agencies/{agencyId}";
     String URI_ENABLE_USERS_WITH_CASELOAD = "/users/add/default/{caseload}";
     String URI_IDENTIFIERS = "/identifiers/{type}/{value}";
     String BOOKING_MOVEMENT = "/bookings/{bookingId}/movement/{seq}";
-    String BOOKING_DETAILS = "/bookings/{bookingId}?basicInfo=true";
```

Comment:
I couldn't see where we add these parameters back in during the client call

Reply:
It's here: https://github.com/ministryofjustice/keyworker-api/pull/229/files#diff-b43233b1d86e65fa1a05b668437219ddR312